src/frontend/komponenter/Felleskomponenter/Skjemasteg/Skjemasteg.tsx

```diff
@@ -38,17 +41,24 @@ const Navigering = styled.div`
     justify-content: space-between;
 `;
 
+const TilbakestillKnapp = styled(Knapp)`
+    margin-left: 20px;
+`;
+
 const Skjemasteg: React.FunctionComponent<IProps> = ({
     children,
     className,
     forrigeKnappTittel,
     forrigeOnClick,
     nesteKnappTittel,
     nesteOnClick,
+    tilbakestillOnClick,
+    tilbakestillKnappTittel,
     senderInn,
     tittel,
     maxWidthStyle = '40rem',
     skalViseNesteKnapp = true,
+    skalViseTilbakestillKnapp,
```

Comment:
Is this used?

Reply:
We renamed it to make it more descriptive